src/js/edu-benefits/1990-rjsf/config/form.js

```diff
@@ -150,21 +162,51 @@ const formConfig = {
           title: 'Service periods',
           path: 'military-history/service-periods',
           uiSchema: {
+            'ui:title': 'Service periods',
+            toursOfDuty: _.merge(toursOfDuty.uiSchema, {
+              'ui:title': null,
+              'ui:description': 'Please record all your periods of service.'
+            })
           },
           schema: {
             type: 'object',
             properties: {
+              toursOfDuty: toursOfDuty.schema(fullSchema1990, {
+                required: ['serviceBranch', 'dateRange.from'],
+                fields: ['serviceBranch', 'serviceStatus', 'dateRange', 'applyPeriodToSelected', 'benefitsToApplyTo', 'view:disclaimer']
+              })
             }
           }
         },
         militaryService: {
           title: 'Military service',
           path: 'military-history/military-service',
           uiSchema: {
+            serviceAcademyGraduationYear: _.assign(yearUI, {
+              'ui:title': 'If you received a commission from a military service academy, what year did you graduate?',
+              'ui:options': {
+                widgetClassNames: 'usa-input-medium'
```

Comment:
Although it is not always the desired class (e.g. ROTC scholarship year/amount field on the next page), it is the default, so this change is still a great idea.